corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -501,18 +519,48 @@ <h4 class="modal-title">{% trans "Get a Demo of CommCare" %}</h4>
               </div>
             </div>
           </div>
+
+
+          <div class="modal fade"
```

Comment:
Can you extract this into a partial? This file is already really long.

Reply:
addressed in https://github.com/dimagi/commcare-hq/pull/22503 otherwise merge conflicts